nemo/collections/tts/waveglow_modules.py

```diff
@@ -161,7 +161,7 @@ def output_ports(self):
         return {"audio": NeuralType(('B', 'T'), AudioSignal(freq=self.sample_rate))}
 
     def __str__(self):
-        return "WaveGlowNM"
+        return "WaveGlowInferNM"
```

Comment:
Is the name change necessary?

Reply:
It helps Tracer to say the truth about what class it fails in.

Reply:
Is it necessary for converting waveglow into an onnx format?
If not, I am changing it back.

Reply:
@blisc yes, please change it back.